spring-cloud-gcp-pubsub-stream-binder/src/main/java/org/springframework/cloud/gcp/stream/binder/pubsub/PubSubMessageChannelBinder.java

```diff
@@ -80,7 +80,13 @@ protected MessageHandler createProducerMessageHandler(ProducerDestination destin
 	protected MessageProducer createConsumerEndpoint(ConsumerDestination destination, String group,
 			ExtendedConsumerProperties<PubSubConsumerProperties> properties) {
 
-		return new PubSubInboundChannelAdapter(this.pubSubTemplate, destination.getName());
+		PubSubInboundChannelAdapter adapter = new PubSubInboundChannelAdapter(this.pubSubTemplate,
+				destination.getName());
+
+		ErrorInfrastructure errorInfrastructure = registerErrorInfrastructure(destination, group, properties);
```

Comment:
I suggest you to override:
```
 protected String errorsBaseName(ConsumerDestination destination, String group, C consumerProperties) {
        return destination.getName() + "." + group + ".errors";
    }
```

to remove that `+ "." + group` from the name for error channel.

This way we won't have that extra `.my-group` in your sample and the name will be consistent with other binder implementations.

Reply:
Thanks, nice suggestion.

Reply:
@dzou @artembilan would be very nice if I had the option to specify the subscription name that I want to use instead of getting a name which is always prefixed with the topic name. I've found this code review while looking for the answer for this in google... maybe you guys can help.

Reply:
@renannprado - Yeah we can definitely look into this more. Could you create a new feature request (or question) in our issue tracker with some more details of what you're looking for? We can continue our discussion there.

Reply:
@dzou sure I'll open an issue to continue there. Thanks!